src/20240104_Multi_Patch/config.ts

```diff
@@ -0,0 +1,49 @@
+import {ConfigFile} from '../../generator/types';
+export const config: ConfigFile = {
+  rootOptions: {
+    configFile: 'src/20240104_Multi_Patch/config.ts',
+    author: 'BGD Labs @bgdlabs',
+    pools: [
+      'AaveV3Ethereum',
+      'AaveV3Polygon',
+      'AaveV3Avalanche',
+      'AaveV3Optimism',
+      'AaveV3Arbitrum',
+      'AaveV3Metis',
```

Comment:
not sure what's the usage of this file exactly, but Metis is not part of this proposal afaik